sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -1396,6 +1399,36 @@ def json_string_to_byte_array(encoded_string):
     """Implements org.apache.beam.sdk.util.StringUtils.jsonStringToByteArray."""
     return unquote_to_bytes(encoded_string)
 
+  def get_default_gcp_region(self):
```

Comment:
Do we test this method? (Tests all seemingly overwrite it.)

Reply:
We test the Java equivalent, but not Python. I suppose we should.

Reply:
I think we should, if we can.

Reply:
Added some unit tests, PTAL.